packages/spec/integration/graphql-mock-server/graphql-mock-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  remoteSchemas: [
+    {
+      url: 'https://api.github.com/graphql',
+      headers: {
+        Authorization: 'bearer <YOUR GITHUB API TOKEN>',
```

Comment:
How about using `process.env.GITHUB_API_TOKEN`? As far as I can see this config is only used in the core mixin and therefore it won't land in the bundle, right?